### PR TITLE
Filter Update

### PR DESCRIPTION
**Pull request by TKDMaster12**
New filter layout to fit many more

### CODE REVIEW

**Comment by TKDMaster12** [comment]
@Mandalorian007 
Display all armour classes on tiles (with the condition some of them are too long, but without it doesn't make sense why there are two number) what do you think?

I redid the layout for the filters 
I added new filters, only the size will search still need to change the backend. 
Let me know what you think of the new design (comments, concerns, etc)

**Comment by TKDMaster12** [comment]
@Mandalorian007 
added alignment filter

found error in search.
I can find zombie by name (good fix), but if I use part of a name to search , the other filters don't work

File: src/main/js/newNpcsFromTemplateForm.tsx [hunk 1]
```diff
@@ -39,8 +48,118 @@ const styles = ({ palette, spacing }: Theme) => createStyles({
...
 
+const sizeSuggestions = [{value: 'Tiny', label: 'Tiny'}, {value: 'Small', label: 'Small' }, {value: 'Medium', label: 'Medium' }, {value: 'Large', label: 'Large' }, {value: 'Huge', label: 'Huge' }, {value: 'Gargantuan', label: 'Gargantuan' }];
+
+const movementSuggestions = [{value: 'Walk', label: 'Walk'}, {value: 'Burrow', label: 'Burrow' }, {value: 'Climb', label: 'Climb' }, {value: 'Fly', label: 'Fly' }, {value: 'Swim', label: 'Swim' }];
+
+const typeSuggestions = [{value: 'Aberration', label: 'Aberration'}, {value: 'Beast', label: 'Beast' }, {value: 'Celestial', label: 'Celestial' }, {value: 'Construct', label: 'Construct' },
+{value: 'Dragon', label: 'Dragon' }, {value: 'Elemental', label: 'Elemental' }, {value: 'Fey', label: 'Fey' }, {value: 'Fiend', label: 'Fiend' }, {value: 'Giant', label: 'Giant' },
+{value: 'Humanoid', label: 'Humanoid' }, {value: 'Monstrosity', label: 'Monstrosity' }, {value: 'Ooze', label: 'Ooze' }, {value: 'Plant', label: 'Plant' }, {value: 'Undead', label: 'Undead' }];
+
+const alignmentSuggestions = [{value: 'Lawful Good', label: 'Lawful Good'}, {value: 'Neutral Good', label: 'Neutral Good' }, {value: 'Chaotic Good ', label: 'Chaotic Good ' },
```
Comment (Mandalorian007) [inline src/main/js/newNpcsFromTemplateForm.tsx]: There is a slightly larger list here:
  ANY_NON_GOOD,
  ANY_NON_LAWFUL,
  ANY_CHAOTIC,
  ANY_EVIL,
  ANY,
  UNALIGNED,
  CHAOTIC_EVIL,
  CHAOTIC_NEUTRAL,
  CHAOTIC_GOOD,
  NEUTRAL_EVIL,
  NEUTRAL,
  NEUTRAL_GOOD,
  LAWFUL_EVIL,
  LAWFUL_NEUTRAL,
  LAWFUL_GOOD

Comment (TKDMaster12) [reply src/main/js/newNpcsFromTemplateForm.tsx]: @Mandalorian007 
Good catch done

**Comment by TKDMaster12** [comment]
@Mandalorian007 
removed condition for AC, just left parenthesis 

I am ready to merge if you are?

**Comment by Mandalorian007** [comment]
@TKDMaster12 go for it!

**Pull request merged**